plugins/dell/fu-self-test.c

```diff
@@ -82,9 +82,8 @@ fu_plugin_dell_tpm_func (gconstpointer user_data)
 	g_autoptr(GPtrArray) devices = NULL;
 
 #ifdef HAVE_GETUID
-	if (tpm_server_running == NULL &&
-	    (getuid () != 0 || geteuid () != 0)) {
-		g_test_skip ("TPM tests require simulated TPM2.0 running or need root access with physical TPM");
+	if (tpm_server_running == NULL) {
+		g_test_skip ("TPM tests require simulated TPM2.0 running");
```

Comment:
this doesn't seem correct to change - it is supposed to work with a real tpm OR with root.

Reply:
Ya, it was more of a way of testing if it was a physical TPM issue or a logic issue. Given the arch virtual TPM failed too I guess I need to debug this a bit more. EOUTOFTIME for today.

Reply:
Worked it out.